creativecloud/scripts/scripts.js

```diff
@@ -116,6 +116,9 @@ const locales = {
   gr_el: { ietf: 'el', tk: 'fnx0rsr.css' }, // Greece (Greek)
 };
 
+// eslint-disable-next-line no-use-before-define
```

Comment:
You should not have to do this. You should follow the `milo-college` example so you can avoid putting eslint-disable in your code.

Reply:
I agree. I'll move the function to `utils.js` like milo-college, thanks for the pointer 👍

Reply:
Moved function to `utils.js`.